File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/response/type/TopExecutedBehaviour.kt
package gaia.sdk.response.type

import gaia.sdk.Uuid
import gaia.sdk.ISO8601
import gaia.sdk.Struct
import gaia.sdk.response.intf.*
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonCreator
import gaia.sdk.request.enumeration.*

data class TopExecutedBehaviour @JsonCreator constructor(
    @JsonProperty("behaviourId") val behaviourId:String? = null, 
    @JsonProperty("behaviourName") val behaviourName:String? = null, 
    @JsonProperty("numberOfExecutions") val numberOfExecutions:Int? = null
)

File: gaia-sdk-javascript/src/graphql/response/type/TopExecutedBehaviour.ts


import {Uuid, ISO8601, Struct} from "../../GaiaClient";
import {Order} from "../../request/enumeration/Order";
import {OrderByField} from "../../request/enumeration/OrderByField";
import {EdgeOrderByField} from "../../request/enumeration/EdgeOrderByField";

export interface TopExecutedBehaviour {
    behaviourId?:string, 
    behaviourName?:string, 
    numberOfExecutions?:number
}